File: codes/Kruskal/Python/KruskalMain.py
import Kruskal as k

graph = k.ALGraph(6)

graph.AddEdge(k.Vertex.A, k.Vertex.B, 9)
graph.AddEdge(k.Vertex.B, k.Vertex.C, 2)
graph.AddEdge(k.Vertex.A, k.Vertex.C, 12)
graph.AddEdge(k.Vertex.A, k.Vertex.D, 8)
graph.AddEdge(k.Vertex.D, k.Vertex.C, 6)
graph.AddEdge(k.Vertex.A, k.Vertex.F, 11)
graph.AddEdge(k.Vertex.F, k.Vertex.D, 4)
graph.AddEdge(k.Vertex.D, k.Vertex.E, 3)
graph.AddEdge(k.Vertex.E, k.Vertex.C, 7)
graph.AddEdge(k.Vertex.F, k.Vertex.E, 13)

graph.ConKruskalMST()
graph.ShowGraphEdgeInfo()
graph.ShowGraphEdgeWeightInfo()

File: codes/Kruskal/C++/KruskalMain.cpp
#include <iostream>
#include "ALGraphKruskalCpp.h"

int main(void)
{
	ALGraph graph(6);

	graph.AddEdge(A, B, 9);
	graph.AddEdge(B, C, 2);
	graph.AddEdge(A, C, 12);
	graph.AddEdge(A, D, 8);
	graph.AddEdge(D, C, 6);
	graph.AddEdge(A, F, 11);
	graph.AddEdge(F, D, 4);
	graph.AddEdge(D, E, 3);
	graph.AddEdge(E, C, 7);
	graph.AddEdge(F, E, 13);

	graph.ConKruskalMST();
	graph.ShowGraphEdgeInfo();
	graph.ShowGraphEdgeWeightInfo();

	graph.GraphDestroy();
	return 0;
}